src/main/java/org/springframework/cloud/deployer/spi/cloudfoundry/CloudFoundryAppDeployer.java

```diff
@@ -15,14 +15,6 @@
  */
 package org.springframework.cloud.deployer.spi.cloudfoundry;
 
-import static java.lang.Integer.parseInt;
```

Comment:
can you please update your import organization settings?... the preexisting code was following the conventions (statics first, empty line before org.springframework, etc)